packages/css-library/src/stylesheets/modules/m-megamenu.scss

```diff
@@ -103,7 +103,7 @@ $marketing-container-height: 380px;
       h3 {
         display: block;
         color: $vads-color-black;
-        font-family: Source Sans Pro, sans serif;
+        font-family: $font-family-sans;
```

Comment:
This file is already importing the variables file, and `$font-farmily-sans` contains the updated `Source Sans Pro Web` naming convention. Renaming this fixes rendered font.